File: VB/reporting-example-export/XtraReport1.vb
Imports DevExpress.XtraReports.UI

Public Class XtraReport1
	Private Sub CategoryNameLabel_BeforePrint(sender As Object, e As System.ComponentModel.CancelEventArgs) Handles CategoryNameLabel.BeforePrint
		Dim categoryNameLabel = CType(sender, XRLabel)
		Dim categoryName = categoryNameLabel.Text.Split("/"c)(0)
		categoryNameLabel.Text = categoryName
		categoryNameLabel.Value = categoryName
	End Sub
End Class

File: CS/reporting-example-export/XtraReport1.cs
using System;
using System.Collections;
using System.ComponentModel;
using System.Drawing;
using DevExpress.XtraReports.UI;

namespace reporting_example_export {
    public partial class XtraReport1 : DevExpress.XtraReports.UI.XtraReport {
        public XtraReport1() {
            InitializeComponent();
        }

        private void CategoryNameLabel_BeforePrint(object sender, CancelEventArgs e) {
            var categoryNameLabel = (XRLabel)sender;
            var categoryName = categoryNameLabel.Text.Split('/')[0];
            categoryNameLabel.Text = categoryName;
            categoryNameLabel.Value = categoryName;
        }
    }
}
